### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by CynthiKite**
docs: updated reference to terminology to IndyKite glossary
docs: updated documentation link
docs: removed indykite.one references

# Checklist

- [ ] `make test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows [commit guidelines](./doc/guides/commit-message.md#commit-message-guidelines)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,40 +27,17 @@ Golang Library packages for [IndyKite Platform](https://indykite.com) Client SDK
 import "github.com/indykite/jarvis-sdk-go"
 ```
 
-In order to access to the platform you must obtain an API key first.
+In order to access to the platform you must first obtain AppAgent credentials.
 This key can be obtained either from the [Admin Console](https://console.indykite.id)
```
Comment (maaland) [inline README.md:31]: "These credentials" instead of "this key", to be consistent with the change above.

File: README.md [hunk 2]
```diff
@@ -20,40 +27,17 @@ Golang Library packages for [IndyKite Platform](https://indykite.com) Client SDK
...
-| Application Agent ID     | ID of the agent which makes the application available for the different calls                    |
-| Tenant ID                | ID of the tenant where the digital twin belongs to. The tenant is belong to an application space |
-| Private Key and Settings | The secret which required to reach the system. Indykite provides the necessary secrets           |
-| Property                 | The digital twin's property (eg.: email, name)                                                   |
-| JWT                      | JSON Web Tokens                                                                                  |
-| Introspect               | A process used to validate the token and to retrieve properties assigned to the token            |
-| Patch property           | Add, change or delete a property of a digital twin                                               |
+- Go version 1.19 or later
+- An AppSpace, App, AppAgent, and AppAgent credentials
+- Service account credentials
```
Comment (maaland) [inline README.md:38]: I don't think a Service Account is strictly necessary to set up before using

**Review by maaland — CHANGES_REQUESTED** [review]
To pass the lint check you need to add the "docs:" prefix to your commit message.

File: README.md (hunk 2 above)
Comment (arxeiss) [reply README.md:38]: Well, AppSpace + App + AppAgent + App Agent credentials might not be needed too

- You need AppAgent credentials if using Identity, Ingest, Knowledge API etc. Basically everything except Config API
- You need Service Account when using Config API

Comment (CynthiKite) [reply README.md:38]: I took what was in the current Go SDK documentation in docs.indykite (https://docs.indykite.com/docs/quick-starts/sdks/go-sdk)

We can just use "Go version 1.19 or later" and take the other items out, or at least specify what credentials are needed for what API. Here and in the doc set (not just for Go SDK, but for the others as well).

Comment (arxeiss) [reply README.md:38]: I'm OK with having some small description here. I was just saying that you don't need both. You might need one, second or both. Depends on the usage

**Review by arxeiss — COMMENTED** [review]
1. Squash your commits
2. fix commit message (see examples https://www.notion.so/indykite/Commit-lint-configuration-8ee3971f7c394a9fb60cbe9a2cc1e0b5)

Then I will aprove it

**Pull request closed without merging**